packages/gestalt/src/Toast.js

```diff
@@ -66,6 +67,24 @@ export default function Toast({
   variant = 'default',
   _dangerouslySetPrimaryAction,
 }: Props): Node {
+  /*
+   *
+   * EXPERIMENT LOGIC START
```

Comment:
this seems excessive for seven lines of code?

Reply:
DONE